validation/exclusions_test.go

```diff
@@ -59,6 +63,12 @@ var exclusions = map[string]map[uint64]bool{
 	},
 }
 
+var silences = map[string]map[uint64]time.Time{
+	"Optimism_Portal_2_Params": {
+		10: time.Unix(1726070401, 0), // mainnet/op silenced until Tue 11 Sep 2024 16:00:01 UTC
```

Comment:
Why don't we access the Granite activation timestamp via the `superchain` Go module?

Reply:
Good idea -- [4226d6f](https://github.com/ethereum-optimism/superchain-registry/pull/515/commits/4226d6f346b13d27d9d5f8d9e657e786aecffe4a)